datafusion/expr/src/expr_fn.rs

```diff
@@ -1228,30 +1229,4 @@ mod test {
             unreachable!();
         }
     }
-
-    #[test]
-    fn encode_function_definitions() {
```

Comment:
I don't think these tests add a lot. A better test is to actually call `encode()` via the expr API / dataframe tests. TODO review if this is done somewhere